app/models/concerns/geocodable.rb

```diff
@@ -6,7 +6,6 @@ def country
     end
 
     def area_address_parts
-      city = send(:city) if respond_to?(:city)
```

Comment:
Not needed any more, since all models (Job & User) that include `Geocodable` have that method defined.